ngraph/frontend/frontend_manager/CMakeLists.txt

```diff
@@ -14,11 +14,24 @@ source_group("src" FILES ${LIBRARY_SRC})
 source_group("include" FILES ${LIBRARY_HEADERS})
 source_group("public include" FILES ${LIBRARY_PUBLIC_HEADERS})
 
-# Create shared library
+# Static library
+
+add_library(${TARGET_NAME}_static STATIC ${LIBRARY_SRC} ${LIBRARY_HEADERS} ${LIBRARY_PUBLIC_HEADERS})
+add_library(ngraph::${TARGET_NAME}::static ALIAS ${TARGET_NAME}_static)
+target_link_libraries(${TARGET_NAME}_static PRIVATE ${CMAKE_DL_LIBS} PUBLIC ngraph)
+target_include_directories(${TARGET_NAME}_static PUBLIC $<BUILD_INTERFACE:${FRONTEND_INCLUDE_DIR}>
+        $<INSTALL_INTERFACE:${FRONTEND_INSTALL_INCLUDE}>)
```

Comment:
we don't need do distinguish install / build interface, since this library is not targeted to be installed